.github/workflows/cd.yml

```diff
@@ -107,7 +107,7 @@ jobs:
         shell: bash
 
       - name: Checkout specified branch
-        uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+        uses: actions/checkout@v4.2.2 # v4.2.2
```

Comment:
Do we have to keep the comments or can we just change them to:

```suggestion
        uses: actions/checkout@v4.2.2
```

Reply:
true 
![vs35fh1ruef51](https://github.com/user-attachments/assets/5aaea8f0-a84d-411b-8444-6780a0db581f)

Reply:
done